src/peft/tuners/lora/layer.py

```diff
@@ -47,6 +48,9 @@ def __init__(self, base_layer: nn.Module, **kwargs) -> None:
         # Mark the weight as unmerged
         self._disable_adapters = False
         self.merged_adapters = []
+        self.use_dora: dict[str, bool] = {}
```

Comment:
I thought `use_dora` is a boolean flag? What are the instances of it being a `dict`?